src/execute/index.js

```diff
@@ -151,9 +169,10 @@ export function buildRequest(options) {
     req.headers.accept = responseContentType
   }
 
-  const combinedParameters = []
+  const combinedParameters = deduplicateParameters([]
     .concat(arrayOrEmpty(operation.parameters)) // operation parameters
     .concat(arrayOrEmpty(path.parameters)) // path parameters
+  )
```

Comment:
As mentioned in #1180, this code would grab the parameters from the operation and the parameters from the path, and merge them into a single array. Because this code is called after `normalizeSwagger`, the operation.parameters array would already have the parameters from the path. This would create duplicate parameters. The duplicate parameters aren't necessarily a problem, but the logic surrounding the ambiguous parameter warning doesn't handle this situation properly.

Reply:
There's a small performance cost to this, but I felt it was much less risky than trying to make normalizeSwagger not pass the path parameters down to the operation, and it was better than not concatenating the arrays here.